Formula/v/virtctl.rb

```diff
@@ -0,0 +1,22 @@
+class Virtctl < Formula
+  desc "Allows for using more advanced kubevirt features"
+  homepage "https://kubevirt.io/"
+  url "https://github.com/kubevirt/kubevirt/archive/refs/tags/v1.1.0.tar.gz"
+  sha256 "3f51390dfcd16de55ba9dd8eac6f587916fdccb979df71003bf26b00de3ee7c4"
+  license "Apache-2.0"
+
```

Comment:
let's also add a head build
```suggestion
  license "Apache-2.0"
  head "https://github.com/kubevirt/kubevirt.git", branch: "main"
```